src/DBus.vala

```diff
@@ -237,6 +237,13 @@ public class Notifications.Server : Object {
         unowned string sound;
 
         switch (category) {
+            case "call":
+            case "call.ended":
+            case "call.incoming":
+            case "call.unanswered":
+                // We probably should use "phone-incoming-call" but it sounds awful in freedesktop theme
```

Comment:
Let's use it anyways and open an issue report against our sound theme to provide a better ring sound :)